src/functions/views/provider.php

```diff
@@ -128,10 +128,11 @@ function tribe_events_widgets_v2_is_enabled() {
 	}
 
 
-	$env_var = getenv( 'TRIBE_EVENTS_WIDGETS_V2_ENABLED' );
+	$env_var = getenv( 'TRIBE_EVENTS_WIDGETS_V2_DISABLED' );
 	if ( false !== $env_var ) {
 		return (bool) $env_var;
 	}
```

Comment:
the logic needs to flip here, since `$env_var = true` would mean V2 widgets is disabled, but we'd be returning `true`, which in the context of this function means widgets v2 is enabled.